app/src/ui/changes/changes-list.tsx

```diff
@@ -26,6 +26,7 @@ interface IChangesListProps {
   readonly onCreateCommit: (message: ICommitMessage) => Promise<boolean>
   readonly onDiscardChanges: (file: WorkingDirectoryFileChange) => void
   readonly onDiscardAllChanges: (files: ReadonlyArray<WorkingDirectoryFileChange>) => void
+  readonly onRevealInFileManager: (path: string) => void
```

Comment:
Can we document this please, specifically with information about how the path parameter is expected to be a path relative to the repository root.